src/stats.js

```diff
@@ -155,12 +123,15 @@ export default async function stats(emitter, upstreamName, context, next) {
       };
     }
 
-    context.res.stats.attempts.push(attempt);
-
-    if (cacheAudit.length > 0) {
-      context.res.stats.cacheAudit.push(cacheAudit);
+    if (context.cacheStatus && Array.isArray(context.cacheStatus)) {
+      context.cacheStatus.forEach((cacheStatus) => {
+        attempt.cache[camelCase(cacheStatus, false, '_')] = true;
```

Comment:
given `_` is the only separator we get, we could update the default in the `camelCase` util so we don't have to pass these extra parameters. What do you think?

Reply:
Yep, makes sense. Updating.

Reply:
Done d261519